src/inputs/mod.rs

```diff
@@ -262,8 +260,8 @@ impl Pump for KafkaPump {
         match self.kafka_consumer.next() {
             TimelyAction::AdvanceTo(epoch) => self.push_to_timely.advance_to(epoch),
             TimelyAction::Emit(item) => Python::with_gil(|py| {
-                let py_any_string = TdPyAny::from(PyString::new(py, &item));
-                self.push_to_timely.send(py_any_string);
+                let key_payload = vec![item.0, item.1].into_py(py);
```

Comment:
Minor point, but does `(item.0, item.1)` work? Using a `Vec` and the resulting Python `list` allows / implies it to be resizable. Also you don't have to allocate a vec.